community-tutorials/first-steps-to-protect-your-linux-server-against-common-attacks/01-en.md

```diff
@@ -0,0 +1,377 @@
+---
+title: First steps to secure your Linux server
+description: A few simple steps to protect your Linux server from common attacks  
+updated_at: 2021-11-10
+slug: first-steps-to-protect-your-linux-server-against-common-attacks
+author_name: Kim Oliver Drechsel
+author_url: https://drechsel.xyz/
+author_image: https://avatars.githubusercontent.com/u/27567533
+author_bio: https://github.com/kimdre/kimdre/blob/main/README.md
```

Comment:
Please update your author_bio. It is not able to paste a link here - just plain text